src/controllers/AuthController.ts

```diff
@@ -0,0 +1,106 @@
+import { Router } from "../modules/Router/Router";
+import { AuthAPI, LoginData, SignupData } from "../services/API/AuthAPI";
+import store from "../modules/Store/Store";
+import { ChatsAPI } from "../services/API/ChatsAPI";
+import chatsController from "./ChatsController";
+import profileController from "./ProfileController";
+class AuthController {
```

Comment:
Будет лучше читаться, если делать отступы
```suggestion
import profileController from "./ProfileController";

class AuthController {
```